src/imap.erl

```diff
@@ -206,6 +206,9 @@ start(ConnSpec, Opts) ->
 %% @equiv start_link(ConnSpec, [])
 -spec start_link(connspec()) ->
     {ok, pid()} | _.
+%% Allow starting with Poolboy
```

Comment:
super nitpick: mind ending the comment with a `.`? The rest of the codebase uses that convention because it has an Erlang-y feel.